web3/queries/votes/getVoteTransactionHashes.ts

```diff
@@ -0,0 +1,19 @@
+import { VotingV2Ethers } from "@uma/contracts-frontend";
+import { goerliDeployBlock } from "constants/deployBlocks";
+import { PriceRequestByKeyT, UniqueKeyT } from "types";
+
+export async function getVoteTransactionHashes(voting: VotingV2Ethers, allVotesByKey: PriceRequestByKeyT) {
+  const filter = voting.filters.PriceRequestAdded(null, null, null);
+  const events = await voting.queryFilter(filter, goerliDeployBlock);
```

Comment:
Not sure about this but I don't know if node providers like infura, in ethereum, have a block range limit to query event logs
`await voting.queryFilter(filter, goerliDeployBlock)`